Builder crash

### Minecolonies version
develop branch

### Expected behavior
Builder will build a building.

### Actual behaviour
Server crash (only on remote)

```

net.minecraft.util.ReportedException: Ticking entity
        at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:707) ~[MinecraftServer.class:?]
        at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:344) ~[ko.class:?]
        at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:605) ~[MinecraftServer.class:?]
        at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:481) [MinecraftServer.class:?]
        at java.lang.Thread.run(Thread.java:745) [?:1.8.0_101]
Caused by: java.lang.NoSuchMethodError: net.minecraft.block.Block.func_149648_K()Z
        at com.minecolonies.util.BlockUtils.getItemStackFromBlockState(BlockUtils.java:130) ~[BlockUtils.class:?]
        at com.minecolonies.entity.ai.citizen.builder.EntityAIStructureBuilder.placeBlock(EntityAIStructureBuilder.java:721) ~[EntityAIStructureBuilder.class:?]
        at com.minecolonies.entity.ai.citizen.builder.EntityAIStructureBuilder.placeBlockAt(EntityAIStructureBuilder.java:580) ~[EntityAIStructureBuilder.class:?]
        at com.minecolonies.entity.ai.citizen.builder.EntityAIStructureBuilder.structureStep(EntityAIStructureBuilder.java:507) ~[EntityAIStructureBuilder.class:?]
        at com.minecolonies.entity.ai.util.AITarget.apply(AITarget.java:122) ~[AITarget.class:?]
        at com.minecolonies.entity.ai.basic.AbstractAISkeleton.applyTarget(AbstractAISkeleton.java:188) ~[AbstractAISkeleton.class:?]
        at com.minecolonies.entity.ai.basic.AbstractAISkeleton.checkOnTarget(AbstractAISkeleton.java:171) ~[AbstractAISkeleton.class:?]
        at java.util.stream.MatchOps$1MatchSink.accept(MatchOps.java:90) ~[?:1.8.0_101]
        at java.util.ArrayList$ArrayListSpliterator.tryAdvance(ArrayList.java:1351) ~[?:1.8.0_101]
        at java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:126) ~[?:1.8.0_101]
        at java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:498) ~[?:1.8.0_101]
        at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:485) ~[?:1.8.0_101]
        at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:471) ~[?:1.8.0_101]
        at java.util.stream.MatchOps$MatchOp.evaluateSequential(MatchOps.java:230) ~[?:1.8.0_101]
        at java.util.stream.MatchOps$MatchOp.evaluateSequential(MatchOps.java:196) ~[?:1.8.0_101]
        at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234) ~[?:1.8.0_101]
        at java.util.stream.ReferencePipeline.anyMatch(ReferencePipeline.java:449) ~[?:1.8.0_101]
        at com.minecolonies.entity.ai.basic.AbstractAISkeleton.func_75246_d(AbstractAISkeleton.java:138) ~[AbstractAISkeleton.class:?]
        at net.minecraft.entity.ai.EntityAITasks.func_75774_a(SourceFile:102) ~[re.class:?]
        at net.minecraft.entity.EntityLiving.func_70626_be(EntityLiving.java:559) ~[ps.class:?]
        at net.minecraft.entity.EntityLivingBase.func_70636_d(EntityLivingBase.java:1775) ~[pr.class:?]
        at net.minecraft.entity.EntityLiving.func_70636_d(EntityLiving.java:382) ~[ps.class:?]
        at net.minecraft.entity.EntityAgeable.func_70636_d(EntityAgeable.java:146) ~[ph.class:?]
        at com.minecolonies.entity.EntityCitizen.func_70636_d(EntityCitizen.java:482) ~[EntityCitizen.class:?]
        at net.minecraft.entity.EntityLivingBase.func_70071_h_(EntityLivingBase.java:1614) ~[pr.class:?]
        at net.minecraft.entity.EntityLiving.func_70071_h_(EntityLiving.java:228) ~[ps.class:?]
        at net.minecraft.world.World.func_72866_a(World.java:1862) ~[adm.class:?]
        at net.minecraft.world.WorldServer.func_72866_a(WorldServer.java:700) ~[le.class:?]
        at net.minecraft.world.World.func_72870_g(World.java:1831) ~[adm.class:?]
        at net.minecraft.world.World.func_72939_s(World.java:1663) ~[adm.class:?]
        at net.minecraft.world.WorldServer.func_72939_s(WorldServer.java:545) ~[le.class:?]
        at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:701) ~[MinecraftServer.class:?]
        ... 4 more
```

### Steps to reproduce the problem
Play the game on a server and let the builder build

@cltnschlosser can we please fix this :laughing:

--- a/src/main/java/com/minecolonies/util/BlockUtils.java
+++ b/src/main/java/com/minecolonies/util/BlockUtils.java
@@ -127,7 +127,7 @@ public final class BlockUtils
         }
 
         Block block = blockState.getBlock();
-        if (item instanceof ItemBlock && !block.isFlowerPot())
+        if (item instanceof ItemBlock)
         {
             block = Block.getBlockFromItem(item);
         }

